### PR TITLE
Remove discontinued iBeacons; add one iBeacon

### PR DESCRIPTION
**Pull request by BlueCharmBeacons**
## Proposed change
Remove links to two discounted Blue Charm beacons and replace with a more current Blue Charm item

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by bdraco — DISMISSED** [review]

**Comment by bdraco** [comment]
Thanks @BlueCharmBeacons

File: source/_integrations/ibeacon.markdown [hunk 1]
```diff
@@ -61,8 +61,7 @@ The integration will create an Estimated Distance sensor by default. This estima
 
 - [Blue Charm Beacons BC011-MultiBeacon](https://bluecharmbeacons.com/product/bluetooth-ble-multi-beacon-bc011/)
 - [Blue Charm Beacons BC021-MultiBeacon](https://bluecharmbeacons.com/product/bluetooth-ble-ibeacon-bc021-multibeacon-with-button-trigger-and-motion-sensor/)
-- [Blue Charm Beacons BC037G-GeoPattern-iBeacon](https://bluecharmbeacons.com/product/blue-charm-bc037-ibeacon/)
-- [Blue Charm Beacons BC037S-SmoothPattern-iBeacon](https://bluecharmbeacons.com/product/bluetooth-ble-ibeacon-bc037s-ibeacon/)
+- [Blue Charm Beacons BC-U1-USB-Powered-MultiBeacon](https://bluecharmbeacons.com/product/bluetooth-ble-ibeacon-bc-u1-multibeacon-usb-powered/)
```
Comment (bdraco) [inline source/_integrations/ibeacon.markdown:64]:
```suggestion
- [Blue Charm Beacons BC-U1-USB-Powered-MultiBeacon](https://bluecharmbeacons.com/product/bluetooth-ble-ibeacon-bc-u1-multibeacon-usb-powered/)
- Blue Charm Beacons BC037G-GeoPattern-iBeacon (discontinued)
- Blue Charm Beacons BC037S-SmoothPattern-iBeacon (discontinued)
```

I think it makes sense to tell them they still work though even if they are discontinued?

Comment (BlueCharmBeacons) [reply source/_integrations/ibeacon.markdown:64]: Yes, that is fine. Thanks. First ever pull request!

File: source/_integrations/ibeacon.markdown [hunk 2]
```diff
@@ -61,8 +61,9 @@ The integration will create an Estimated Distance sensor by default. This estima
...
+- Blue Charm Beacons BC037G-GeoPattern-iBeacon (discontinued)
+- Blue Charm Beacons BC037S-SmoothPattern-iBeacon (discontinued)
 - [Blue Charm Beacons BC08-MultiBeacon](https://bluecharmbeacons.com/product/blue-charm-beacons-bluetooth-ble-ibeacon-bc08-multibeacon-w-motion-sensor-and-button-trigger-ble-5-0/)
```
Comment (bdraco) [inline source/_integrations/ibeacon.markdown]:
```suggestion
- [Blue Charm Beacons BC08-MultiBeacon](https://bluecharmbeacons.com/product/blue-charm-beacons-bluetooth-ble-ibeacon-bc08-multibeacon-w-motion-sensor-and-button-trigger-ble-5-0/)
- Blue Charm Beacons BC037G-GeoPattern-iBeacon (discontinued)
- Blue Charm Beacons BC037S-SmoothPattern-iBeacon (discontinued)
```

**Review by bdraco — APPROVED** [review]

**Pull request merged**